spec/ndx-pose.extensions.yaml

```diff
@@ -92,27 +121,87 @@ groups:
       dtype: text
       doc: Version string of the software tool used.
       required: false
-  - name: nodes
+  groups:
+  - neurodata_type_inc: PoseEstimationSeries
+    doc: Estimated position data for each body part.
+    quantity: '*'
+  links:
+  - target_type: Skeleton
+    doc: Layout of body part locations and connections.
+- neurodata_type_def: TrainingFrame
+  neurodata_type_inc: NWBDataInterface
+  default_name: TrainingFrame
+  doc: Group that holds ground-truth position data for all instances of a skeleton in a single frame.
+  attributes:
+  - name: annotator
     dtype: text
-    dims:
-    - num_body_parts
-    shape:
-    - null
-    doc: Array of body part names corresponding to the names of the PoseEstimationSeries
-      objects within this group.
+    doc: Name of annotator who labeled the TrainingFrame.
+    required: false
+  groups:
+  - neurodata_type_inc: Instance
+    doc: Position data for a single instance of a skeleton in a single training frame.
+    quantity: '*'
+  - name: source_video
+    neurodata_type_inc: ImageSeries
+    doc: Path to original video file and frame used.
     quantity: '?'
-  - name: edges
+    attributes:
+    - name: path
```

Comment:
`ImageSeries` already supports external storage of videos using its `external_file` dataset. So I think `path` is unnecessary.

Reply:
I removed the path attribute.